src/Symfony/Component/Console/Helper/QuestionHelper.php

```diff
@@ -122,7 +136,7 @@ public function getName()
      * @param OutputInterface $output
      * @param Question        $question
      *
-     * @return bool|mixed|null|string
+     * @return mixed
```

Comment:
I agree. I still feel it better to know what it return ^^. Thank you @Ener-Getick